### ISSUE TITLE
Your current user or role does not have access to Kubernetes objects on this EKS cluster

### ISSUE THREAD

**Issue by parkand1** [shapirov103/cdk-eks-blueprint#42]
When users deploy the EKS cluster based on the instructions in the README they may run into the following error - "Your current user or role does not have access to Kubernetes objects on this EKS cluster" My recommendation is creating a prerequisites step to ensure the right IAM roles/polices are in place

**Comment by shapirov103** [comment]
Do you mean console access? If yes, then my recommendation is to do the following:
1. Allow a configuration option for the blueprint to provide console cluster access for current user. 
2. If option is provided then automatically detect whether the stack is provisioned under a specific role or IAM user and add console access permissions.

The other option that I working on right now is enabling cluster access (kubectl atm) for team users. We could extend this with console access as well as an option.

**Comment by parkand1** [comment]
Yes, I think that would be helpful if we could extend this for console access as well
